### DISCUSSION TITLE
Antialised font is bulkier on qutebrowser

### DISCUSSION THREAD

**Discussion by duarm** [qutebrowser/qutebrowser#7626] [Q&A] [answered]
I'm using a Terminus which works best not antialiased, I want to disable the antialiasing for qutebrowser only, and only for qutebrowser's interface, not for the website content.

Usually the program provides a format to specify this information, the pango format is very common, something like this: `"Terminus:pixelsize=22:antialias=false:autohint=true"`, how to do this in qutebrowser?

I've tried
```
c.fonts.default_family = "Terminus antialias=false"
```

and other variations, searched the full settings documentation, is there a way?

**Comment by toofar** [comment]
> Usually the program provides a format to specify this information, the pango format is very common, something like this: "Terminus:pixelsize=22:antialias=false:autohint=true", how to do this in qutebrowser?

I think it uses the same syntax? I believe we/Qt pass of to fontconfig (oh, what platform are you on!).

**Reply by duarm** [reply]
Linux,  I've tried this
```
c.fonts.default_family = "Terminus;antialias=false"
```

looking closer now, I think the font is not antialised, it's working, but look at the comparison between the same non antialised font on dwm and on qutebrowser

![image](https://user-images.githubusercontent.com/26150581/226135926-9fad942f-7012-4b6a-aeeb-58e18b9e8426.png)

qutebrowser's one is bulkier, what is the problem here?

**Reply by toofar** [reply]
Try with a colon, not semicolon. I'm just using fontconfig tools to confirm that it picks up the attribute (eg `fc-pattern "Terminus:antialias=false"`). I don't know anything about fonts so that's about all I can do.

**Reply by duarm** [reply]
it works with a semicolon, not with a colon, not sure why tho, thanks for help, the antialiasing is working, but the new problem is that the font is bulkier, I'll change title.

**Reply by duarm** [reply]
I'm dumb, the font is bold

**Comment by duarm** [comment, marked as answer]
The solution was that I needed to use semicolon, not a colon
```
c.fonts.default_family = "Terminus;antialias=false"
```
and the font is bulkier because the font was bold.
